python/tvm/relay/frontend/onnx.py

```diff
@@ -2199,6 +2199,241 @@ def body_fn(*loop_inputs):
         return outputs
 
 
+class NonMaxSuppression(OnnxOpConverter):
+    """Operator converter for NonMaxSuppression."""
+
+    @classmethod
```

Comment:
A quick description of why we need three loops to implement onnx NMS would be helpful for readers trying to make sense of what's going on here.